Makefile

```
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.2.3
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.13.9
```

```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 1.13
```
so that we can have the patch version updated